verify.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
I wouldn't necessarily assume it does. I'm not sure how those kind of options are inherited by scripts.